usbreset.c

```diff
@@ -125,7 +131,9 @@ static struct usbentry *find_device(int *bus, int *dev, int *vid, int *pid,
 	while ((e = parse_devlist(devs)) != NULL)
 		if ((bus && (e->bus_num == *bus) && (e->dev_num == *dev)) ||
 			(vid && (e->vendor_id == *vid) && (e->product_id == *pid)) ||
-			(product && !strcasecmp(e->product_name, product))) {
+			(product_or_serial && !strcasecmp(e->serial, product_or_serial)) ||
+			(product_or_serial && !strcasecmp(e->product_name, product_or_serial))
+			) {
```

Comment:
Odd coding style change, can you fix this up please?

Reply:
I squashed the commits in faeaa86.